tests/testthat/_snaps/learn.md

```diff
@@ -370,14 +404,14 @@
       # A tibble: 8 x 5
         child_table child_fk_cols parent_table parent_key_cols on_delete
         <chr>       <keys>        <chr>        <keys>          <chr>    
-      1 disps       account_id    accounts     id              no_action
-      2 loans       account_id    accounts     id              no_action
-      3 orders      account_id    accounts     id              no_action
-      4 trans       account_id    accounts     id              no_action
-      5 disps       client_id     clients      id              no_action
-      6 cards       disp_id       disps        id              no_action
-      7 accounts    district_id   districts    id              no_action
-      8 clients     district_id   districts    id              no_action
+      1 disps       account_id    accounts     id              cascade  
+      2 loans       account_id    accounts     id              cascade  
+      3 orders      account_id    accounts     id              cascade  
+      4 trans       account_id    accounts     id              cascade  
+      5 disps       client_id     clients      id              cascade  
+      6 cards       disp_id       disps        id              cascade  
+      7 accounts    district_id   districts    id              cascade  
+      8 clients     district_id   districts    id              cascade  
```

Comment:
I noticed that all the `on_delete` columns here have changed from `no_action` to `cascade`. I need to investigate if this was as intented.

Reply:
I forgot that in the main branch it always defaults to no_action. I double checked from the Financial_ijs database to confirm. All good here.
![image](https://user-images.githubusercontent.com/19807339/236195165-af24fb6d-a431-4b2a-bb62-5d579a9a5ed7.png)